src/l500/ac-trigger.cpp

```diff
@@ -995,11 +1013,19 @@ namespace ivcam2 {
                     settings.hum_temp = _temp;
                     settings.ambient = _ambient;
                     settings.receiver_gain = _receiver_gain;
-                    depth_to_rgb_calibration algo( settings,
-                                                   df, irf,
-                                                   _cf, _pcf, _last_yuy_data,
-                                                   cal_info, cal_regs,
-                                                   should_continue );
+                    depth_to_rgb_calibration algo(
+                        settings,
+                        df,
+                        irf,
+                        _cf,
+                        _pcf,
+                        _last_yuy_data,
+                        cal_info,
+                        cal_regs,
+                        read_intrinsics_from_camera( _dev, _cf.get_profile() ),
+                        _temp,
```

Comment:
Shouldn't be needed -- it's in `settings`

Reply:
done